ts/src/base/Exchange.ts

```diff
@@ -1542,6 +1542,35 @@ export default class Exchange {
         };
     }
 
+    currencyStructure () {
+        return {
+            'info': undefined,
+            'id': undefined,
+            'numericId': undefined,
+            'code': undefined,
+            'precision': undefined,
+            'type': undefined,
+            'margin': undefined,
+            'name': undefined,
+            'active': undefined,
+            'deposit': undefined,
+            'withdraw': undefined,
+            'fee': undefined,
+            'fees': undefined,
+            'networks': undefined,
```

Comment:
@samgermain I think we should have there `{}` even empty, like we have for `limits` down below